models/articles-model.js

```diff
@@ -1,12 +1,25 @@
 const db = require("../db/connection");
 
 exports.selectArticlesById = (params) => {
-  let query = `SELECT * FROM articles`;
+  const id = params.article_id;
 
-  query += ` WHERE article_id = ${params.article_id}`;
-  query += ";";
+  if (isNaN(+id)) {
+    return Promise.reject({ code: 400, msg: "Bad Request", custom: true });
+  }
+
```

Comment:
You don't need to do the above, if the id is not a number PSQL will throw an error for you when it attempts the query!